src/data_hub/_api.py

```diff
@@ -76,6 +81,9 @@ def _get_default_client(client=None, refresh: bool = True):
             base_url=base_url,
         )
 
+        if debug:
+            client = client.with_raw_response
```

Comment:
This feels like a complicated way to make the behavior conditional on `debug=True`.

Reply:
fixed